src/main/kotlin/org/rust/lang/core/psi/ext/PsiElement.kt

```diff
@@ -209,12 +209,24 @@ inline fun <reified T : PsiElement> PsiElement.descendantsWithMacrosOfType(): Co
 val PsiElement.contextualFile: PsiFile
     get() = contextOrSelf() ?: error("Element outside of file: $text")
 
+/**
+ * Finds first sibling that is not whitespace before given element.
+ */
+fun PsiElement?.getPrevNonWhitespaceSibling(): PsiElement? =
```

Comment:
Please move this method and the similar below to the intention class because it is used only there